sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/arithmetic.scala

```diff
@@ -448,7 +448,7 @@ case class IntegralDivide(left: Expression, right: Expression) extends DivModLik
   usage = "expr1 _FUNC_ expr2 - Returns the remainder after `expr1`/`expr2`.",
   examples = """
     Examples:
-      > SELECT 2 _FUNC_ 1.8;
```

Comment:
Ur, I know the intention, but this is a revert of `[SPARK-29237][SQL] Prevent real function names in expression example template`.

Reply:
Unfortunately, using of the `_FUNC_` template is impossible here because the expression has 2 function names `%` and `mod` but one of the example always causes an error:
```sql
spark-sql> SELECT 2 mod 1.8;
Error in query: 
extraneous input '1.8' expecting <EOF>(line 1, pos 13)

== SQL ==
SELECT 2 mod 1.8
-------------^^^
```
```sql
spark-sql> SELECT %(2, 1.8);
Error in query: 
no viable alternative at input 'SELECT %'(line 1, pos 7)

== SQL ==
SELECT %(2, 1.8)
-------^^^
```

Reply:
I thought `blacklisting` is a better idea for this. 
And, the blacklisting was in this PR already before, isn't it?

Reply:
hmm, probably I don't understand you. Do you want to show not working example to users? 
For `MOD`, the examples (after replacing `_FUNC_` by `MOD`) are:
```sql
    Examples:
      > SELECT 2 MOD 1.8;
       0.2
      > SELECT MOD(2, 1.8);
       0.2
```
where the first one is incorrect.
For `%`, both are correct:
```sql
    Examples:
      > SELECT 2 % 1.8;
       0.2
      > SELECT MOD(2, 1.8);
       0.2
```
Using `_FUNC_` for the first example, or for the second one, or for both produce always produce an incorrect example. For this particular examples, we cannot use `_FUNC_` at all.